source/sonata_tech.rst

```diff
@@ -576,3 +576,23 @@ Required fields for ``Spykfunc``:
 .. _specification: https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md
 .. _enumeration: https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md#nodes---enum-datatypes
 .. _Tsodyks Markram Model: https://www.pnas.org/content/94/2/719
+
+Format of the electrodes_file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The format of the weights file is described below:
+
+.. table::
+
+  ===================================== ======================= ========== =================== ============= ==========================================
+  Group                                 Field                   Type       Shape               Requirement   Description
+  ===================================== ======================= ========== =================== ============= ==========================================
+  /electrodes/contacts/{electrodename}  layer                   utf8       1                   Optional      Layer of the circuit in which {electrodename} is located. If the electrode is in a region without cortical layers, then "NA". If the electrode is outside the brain, then "Outside"
+  /electrodes/contacts/{electrodename}  position                float32    3                   Mandatory     Position of {electrodename} in microns, in cartesian coordinates
+  /electrodes/contacts/{electrodename}  region                  utf8       1                   Optional      Region in which {electrodename} is located
+  /electrodes/contacts/{electrodename}  type                    utf8       1                   Optional      Either EEG or LFP
+  /electrodes/{population_name}         data                    float64    Total_seg x N_elec  Mandatory     Scaling factor for each segment in the corresponding neuron, in mV/nA
+  /{population_name}                    node_ids                uint64     N_nodes             Mandatory     List of node ids
+  /{population_name}                    sec_ids                 uint64     Total_seg           Mandatory     For each segment, index of the section to which it belongs. Ordered by section ids and grouped by nodes.
```

Comment:
Can `sec_ids` have not be shortened? `section_ids` would more closely match how similar ones are named in the edges files.

Reply:
i could change that